### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by martygrant** [martygrant/uwsgamedevbot#12]
Setup a continuous integration system which will help check new commits build properly and make it easier for other people to deploy changes to the live system.

**Comment by Medallyon** [comment]
Check out https://devcenter.heroku.com/articles/github-integration

**Comment by martygrant** [comment]
Yeah it automatically deploys pushes to master but I was thinking along the lines of having a dev and release branch and setting up a test suite so if other people contribute it can all be automated without me having to step in and check and merge pull requests.
